packages/lib/src/vis/line/LineVis.tsx

```diff
@@ -152,14 +153,33 @@ function LineVis(props: Props) {
                 {`${abscissaLabel ?? 'x'} = ${formatTooltipVal(abscissa)}`}
 
                 <div className={styles.tooltipValue}>
-                  <strong>{formatTooltipVal(value)}</strong>
-                  {error && ` ±${formatTooltipErr(error)}`}
-                  {dtype && <em>{` (${formatNumType(dtype)})`}</em>}
+                  {auxiliaries.length > 0 && (
+                    <span
+                      className={styles.mark}
+                      style={{ color: curveColor }}
+                    />
+                  )}
+                  <span>
+                    <strong>{formatTooltipVal(value)}</strong>
+                    {error && ` ±${formatTooltipErr(error)}`}
```

Comment:
This displays `0` if `error` is 0 which is unfortunate.

It should display either `±0` or nothing at all.

Reply:
Good catch!